src/controller/java/CHIPDeviceController-JNI.cpp

```diff
@@ -234,6 +234,10 @@ JNI_METHOD(jlong, newDeviceController)(JNIEnv * env, jobject self)
 
     ChipLogProgress(Controller, "newDeviceController() called");
 
+    DeviceLayer::PersistedStorage::KeyValueStoreMgrImpl().InitializeWithObject(keyValueStoreManager);
+    using namespace chip::Mdns;
+    InitializeWithObject(serviceResolver);
```

Comment:
If I try `chip::Mdns::InitializeWithObject(serviceResolver)`, doxygen fails with
```
/Users/austinhsieh/repos/connectedhomeip/src/controller/java/CHIPDeviceController-JNI.cpp:238: error: no matching file member found for 
chip::Mdns::InitializeWithObject(serviceResolver)
Possible candidates:
 'void InitializeWithObject(jobject resolverObject)' at line 106 of file /Users/austinhsieh/repos/connectedhomeip/src/controller/java/MdnsImpl.cpp
```
Not sure why.

Reply:
If doxygen is being troublesome how about

`using ::chip::Mdns::InitializeWithObject;`

Please don't import whole namespaces.